rotkehlchen/chain/ethereum/decoding/decoder.py

```diff
@@ -388,7 +394,7 @@ def _maybe_decode_simple_transactions(
                     notes=f'Burned {eth_burned_as_gas} ETH in gas from {location_label}',
                     event_type=HistoryEventType.SPEND,
                     event_subtype=HistoryEventSubType.FEE,
-                    counterparty='gas',
```

Comment:
There is a lot more places where this needs to be changed all over the code.

Also there is other uses of `XDAI` that should have been replaced by the constant.

The reason I did not do it myself is that it's not just making a constant, but using it all over the code where it needs to be used, which is something that would take more time.